libs/designsystem/src/lib/components/segmented-control/segmented-control.component.ts

```diff
@@ -78,6 +79,8 @@ export class SegmentedControlComponent {
     return this._value;
   }
 
+  @Output() selectedIndexChange = new EventEmitter<number>();
```

Comment:
Just out of curiosity, why is this new output needed? 

We should remember to document it in the cookbook too! 😄